### ISSUE TITLE
Doesn't work at all.

### ISSUE THREAD

**Issue by Alivender** [schultzcole/FVTT-Autocomplete-Inline-Properties#16]
Hi,
I've tried this today in Foundry 0.7.9, DnD5e 1.2.4, and with not other modules and it doesn't do anything. I tried typing in the damage formula, in the active effects window, and in the chat. The @ icon doesn't appear and the list of properties doesn't appear either.

![image](https://user-images.githubusercontent.com/74206009/115153305-1dfe0c00-a06d-11eb-8907-d723a0727ddc.png)

**Comment by schultzcole** [comment]
When you first start up the game with only AIP enabled, can you open up the dev tools (F12), then tell me if any errors show up in red in the console? I suspect something is erroring out early on that is preventing the rest of the module from loading.

**Comment by Alivender** [comment]
Hi, thanks for replying.

I am not seeing any errors I'm afraid.
![image](https://user-images.githubusercontent.com/74206009/115270703-819f3c80-a134-11eb-9573-ec7bb1c2735c.png)

**Comment by schultzcole** [comment]
Hmm, that's very strange. Are you using the native foundry app or a browser? If you're using a browser, which one? And what operating system?

**Comment by Alivender** [comment]
It's the native app.

**Comment by Mperor** [comment]
Hello. I'm currently having the same issue but on Firefox 87.0 and Win10 64 as well.

**Comment by schultzcole** [comment]
I'm at a bit of a loss as to what would be causing this issue. In the console logs area can you double check that you see the message `"AIP | Setting up Autocomplete Inline Properties"` probably somewhere near the top?

After that, could I ask you to enable the full debug logging for AIP by opening up `<user data>/modules/autocomplete-inline-properties/aip-setup.mjs` in a text editor and on line 9 change `CONFIG.debug.aip = false;` to `CONFIG.debug.aip = true;` and then reload your world and check the logs again? After enabling the logging you should be able to see a few new logs after the "Setting up Autocomplete Inline Properties" line that say "Registering hook for [...]".

**Comment by schultzcole** [comment]
Ah! I think I know what is going on here. There is currently a limitation that AIP won't work with items that are not owned by an actor. Can you both verify whether you are attempting to use it on an item that is owned by an actor or not?

This limitation should be possible to work around once core 0.8 is released.

I'll update the readme and the package listing on the Foundry website to make this more clear.

**Comment by Mperor** [comment]
Yes! I think you're right. Specifically, I made an item, AIP didn't work on it.
I gave that item to an Actor, then edited that Item from the Char Sheet, and it worked.
That's great to know that I can work around the problem... and the fact that I won't be a problem at all soon.
Thanks!

**Comment by Alivender** [comment]
My apologies for the late reply.

Yes, that was it. I kept testing with items not owned by anybody. I tried with a weapon equipped by one of the players and the autofill options appeared as promised and worked great.

I understand the limitation can work with this. This module is the best source of knowledge on the player stat syntax I could find anywhere, so I'm just so happy I can use it.

Many thanks.
